solidity/revert.sol

```diff
@@ -7,7 +7,7 @@ contract C {
     state = x;
   }
 
-  function echidna_revert() public returns (bool) {
+  function solidity_revert() public returns (bool) {
```

Comment:
was this supposed to change to this new prefix?

Reply:
My mistake!